Secure-Software-Development-Policy/Zowe-SSDP-Organization/Policy.md

```diff
@@ -0,0 +1,206 @@
+# Zowe Secure Software Development (Z-SSD) - Organization Policy (OP) 
+
+>Table of contents:
+>
+>- [Introduction](#introduction)
+>- [Prepare the organization (PO)](#po-prepare-the-organization)
+>- [Protect the Software (PS)](#ps-protect-the-software)
+
+
+##Introduction
+This document contains the Zowe Secure Software Development Organizational Policy (ZSSD-OP).
+
+The organizational policy is intended to set up requirements and govern the secure best practices for the environment (organizational level) in which the requirements set by the [Secure SDLC policy](../Zowe-SSDP-SDLC/Policy.md) can be fulfilled.
+
+Sometimes requirements from OP and DP can interfere or be complementary. Duplications will not be resolved for completeness and clarity of the respective policy.  
```

Comment:
#RE-DEFINE
If we have conflicts between OP and DP, we need to resolve them otherwise how can work get done?